keyboards/lily58/keymaps/rubangarmon/keymap.c

```diff
@@ -0,0 +1,238 @@
+ *                   |      |      |      |/       /         \      \ |      |      |      |
+ *                   `----------------------------'           '------''--------------------'
+  */
+    [_ADJUST] = LAYOUT(
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, QK_BOOT, DB_TOGG, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, QWERTY , XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+                               XXXXXXX, XXXXXXX, _______, XXXXXXX, XXXXXXX,  _______, XXXXXXX, XXXXXXX
+    )
+
+  /* Game
+  * ,-----------------------------------------.                    ,-----------------------------------------.
+  * | Esc  |   1! |   2@ |   3# |   4$ |   5% |                    |   6^ |   7& |   8* |   9( |   0) |BackSP|
+  * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+  * | Tab  |   Q  |   W  |   E  |   R  |   T  |                    |   Y  |   U  |   I  |   O  |   P  |      |
+  * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+  * | CLck |   A  |   S  |   D  |   F  |   G  |-------.    ,-------|   H  |   J  |   K  |   L  |  ;*  |  '   |
+  * |------+------+------+------+------+------|BackSP |    |QWERTY |------+------+------+------+------+------|
+  * | Shft |   Z  |   X  |   C  |   V  |   B  |-------|    |-------|   N  |   M  |   ,  |   .  |   /  |RShift|
+  * `-----------------------------------------/       /     \      \-----------------------------------------'
+  *                   | Ctrl | LAlt | Lower| /Space  /       \Enter \  |Raise | RAlt | LGUI |
+  *                   |      |      |      |/       /         \      \ |      |      |      |
+  *                   `----------------------------'           '------''--------------------'
+    [_GAME] = LAYOUT(
+    KC_ESC,   KC_1,   KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,  KC_8   , KC_9,    KC_0,    KC_BSPC,
+    KC_TAB,   KC_Q,   KC_W,    KC_E,    KC_R,    KC_T,                      KC_Y,    KC_U,  KC_I   , KC_O,    KC_P,    XXXXXXX,
+    KC_CAPS,  KC_A,   KC_S,    KC_D,    KC_F,    KC_G,                      KC_H,    KC_J,  KC_K   , KC_L,    FUCSCLN, KC_QUOT,
+    KC_LSFT,  KC_Z,   KC_X,    KC_C,    KC_V,    KC_B, KC_BSPC,  QWERTY ,   KC_N,    KC_M,  KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
+                            KC_LCTL, KC_LALT,  LOWER , KC_SPC ,  KC_ENT ,  RAISE, KC_RALT,  KC_LGUI
+    )    
+  */
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
```

Comment:
The [tri-layer feature](https://docs.qmk.fm/#/feature_tri_layer) should be used here instead.
```suggestion
```